Microsoft.Research/ClousotMain/Caching/CacheManager.cs

```diff
@@ -1121,7 +1121,8 @@ private void InstallAssumes(InferredExpr<Field, Method> inferredExpr, Method met
 
                     this.ReplayOutput.WriteLine("The reason why the deserialization failed is {0}", e.Message);
                     this.ReplayOutput.WriteLine(
-                        "We remove the incorrect entry from the cache and we replace it with a new one. If the error persists, please report the tool output to logozzo@microsoft.com");
+                        "We remove the incorrect entry from the cache and we replace it with a new one. "
+                        + "If the error persists, please report the tool output at https://github.com/Microsoft/CodeContracts/issues");
```

Comment:
If we're going to change this in the future, maybe it would be useful to extract this path into some constant and not hard-code it in the message?